scripts/components/Interactions/HotspotNavButton.js

```diff
@@ -201,12 +202,28 @@ export default class HotspotNavButton extends React.Component {
 
     const iconSize = clamp(20, Math.min(this.state.sizeWidth / 2, this.state.sizeHeight / 2), 40);
 
+    // Resize hotspot for full screen in static scene based on wrapper
+    let width = this.state.sizeWidth;
+    let height = this.state.sizeHeight;
+
+    if (this.props.staticScene && !this.context.extras.isEditor) {
+      if (staticWidth && staticWidth> 0 && !isNaN(staticWidth))
+      {
+        width = (staticWidth/100)*((this.state.sizeWidth / staticSceneWidth)*100); 
+      }
+  
+      if (staticHeight && staticHeight> 0 && !isNaN(staticHeight))
+      {
+        height = (staticHeight/100)*((this.state.sizeHeight / staticSceneHeight)*100);
+      }
```

Comment:
Litt luft gjør koden litt enklere å lese.
```suggestion
      if (staticWidth && staticWidth > 0 && !isNaN(staticWidth)) {
        width = (staticWidth/100) * ((this.state.sizeWidth / staticSceneWidth) * 100); 
      }
      if (staticHeight && staticHeight > 0 && !isNaN(staticHeight)) {
        height = (staticHeight/100) * ((this.state.sizeHeight / staticSceneHeight) * 100);
      }
```